src/tablecloth/api.clj

```diff
@@ -50,8 +50,6 @@
                          select-columns
                          drop-columns
                          rename-columns
-                         add-or-replace-column
```

Comment:
As we discussed on Zulip. We move `deprecated` var to a `columns` namespace and refer it here